doc/exec_council/ceph-exec-council-sage-responsibilities.rst

```diff
@@ -0,0 +1,254 @@
+.. _ceph-exec-council-sage-responsibilities:
+
```

Comment:
This should have a big warning at the top that this is an archived document that Sage made. (at least as long as we keep it here.)

Reply:
I'll take care of it when I get back to the office.

Reply:
I've added the warning.